### PR TITLE
Changes done for adding windows integration script

### PR DESCRIPTION
**Pull request by dikshachauhan-qasource**
## What does this PR do?
Adding windows integration successfully to a policy.

Hi @michalpristas 

Here, are the changes that I have made for above script. Below is the error that we are getting while adding this integration.

**Error logs:**
`level=warning msg="could not add package to policy because of HTTP code is not 200" elapsedTime=20.923285992s err="<nil>" package="{ windows-3324d79a-c85c-4fb5-8c24-67dce028698c Windows default 1f046770-5e8f-11ed-8358-77c1e5cff8c3 true  [{winlog true [{{metrics windows.powershell} true winlog-windows.powershell-3d8fd865-b8cc-4371-a891-5c52e7d05f4f map[period:{1s string}]}] map[] <nil> <nil>}] { windows Windows 1.15.0}}" response="{\"statusCode\":400,\"error\":\"Bad Request\",\"message\":\"Package policy is invalid: inputs.winlog.streams.windows.powershell.vars.preserve_original_event: Preserve original event is required\\ninputs.winlog.streams.windows.powershell.vars.event_id: Event ID is required\"}" retry=7 statusCode=400`

### CODE REVIEW

File: e2e/_suites/fleet/testresources/windows.json [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+				 "data_stream":{
+                        "type":"logs",
+                        "dataset":"windows.forwarded"
+                    },
+					"vars": {
+						"period": {
+						"type": "text",
+						"value": "10s"
+					},
+					"preserve_original_event": false,
```
Comment (michalpristas) [inline e2e/_suites/fleet/testresources/windows.json:17]:
```suggestion
                                        "preserve_original_event": {
                                            "type":"bool",
                                            "value": false
                                        },
```

File: e2e/_suites/fleet/testresources/windows.json [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+                    "data_stream":{
+                        "type":"logs",
+                        "dataset":"windows.powershell_operational"
+                    },
+                    "vars": {
+						"period": {
+						"type": "text",
+						"value": "10s"
+					},
+					"preserve_original_event": false,
```
Comment (michalpristas) [inline e2e/_suites/fleet/testresources/windows.json:58]:
```suggestion
                                        "preserve_original_event": {
                                            "type":"bool",
                                            "value": false
                                        },
```

File: e2e/_suites/fleet/testresources/windows.json [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+                    "data_stream":{
+                        "type":"logs",
+                        "dataset":"windows.sysmon_operational"
+                    },
+                    "vars": {
+						"period": {
+						"type": "text",
+						"value": "10s"
+					},
+					"preserve_original_event": false,
```
Comment (michalpristas) [inline e2e/_suites/fleet/testresources/windows.json:77]:
```suggestion
                                        "preserve_original_event": {
                                            "type":"bool",
                                            "value": false
                                        },
```

File: e2e/_suites/fleet/testresources/windows.json [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+                    "data_stream":{
+                        "type":"metrics",
+                        "dataset":"windows.perfmon"
+                    },
+                    "vars": {
+						"period": {
+						"type": "text",
+						"value": "10s"
+					},
+					"preserve_original_event": false,
```
Comment (michalpristas) [inline e2e/_suites/fleet/testresources/windows.json:96]:
```suggestion
                                        "preserve_original_event": {
                                            "type":"bool",
                                            "value": false
                                        },
```

File: e2e/_suites/fleet/testresources/windows.json [hunk 5]
```diff
@@ -0,0 +1,125 @@
...
+                    "data_stream":{
+                        "type":"metrics",
+                        "dataset":"windows.service"
+                    },
+                    "vars": {
+						"period": {
+						"type": "text",
+						"value": "10s"
+					},
+					"preserve_original_event": false,
```
Comment (michalpristas) [inline e2e/_suites/fleet/testresources/windows.json:115]:
```suggestion
                                        "preserve_original_event": {
                                            "type":"bool",
                                            "value": false
                                        },
```

**Review by michalpristas — APPROVED** [review]
please also paste `windows.json` into https://jsonlint.com/
to fix formatting and indentations to make it more readable

**Review by michalpristas — CHANGES_REQUESTED** [review]
wrong button

**Comment by dikshachauhan-qasource** [comment]
Hi @michalpristas 

I have reran the script after making the above suggested changes, however the issue still persists as reported.

Could you please recheck and let me know further inputs, that could work.

Thanks in Advance.

File: e2e/_suites/fleet/testresources/windows.json [hunk 6]
```diff
@@ -0,0 +1,143 @@
+{
```
Comment (michalpristas) [inline e2e/_suites/fleet/testresources/windows.json:1]: try this one
```json
{
	"inputs": [{
		"type": "winlog",
		"enabled": true,
		"streams": [
		  {
				"data_stream": {
					"type": "logs",
					"dataset": "windows.forwarded"
				},
				"vars": {
					"period": {
						"type": "text",
						"value": "10s"
					},
					"preserve_original_event": {
						"type": "bool",
						"value": false
					},
					"event_id": {
						"type": "text",
						"value": "some_id"
					},
					"tags": {
						"type": "text",
						"value": ["preserve_original_event", "forwarded", "microsoft_dhcp"]
					}
				},
				"id": "winlog-windows.forwarded-7886c582-c3fa-4ce1-a016-a6819dbba179",
				"enabled": true
			},
			{
				"data_stream": {
					"type": "logs",
					"dataset": "windows.powershell"
				},
				"vars": {
					"period": {
						"type": "text",
						"value": "10s"
					},
					"preserve_original_event": {
						"type": "bool",
						"value": false
					},
					"event_id": {
						"type": "text",
						"value": "some_id"
					},
					"tags": {
						"type": "text",
						"value": ["preserve_original_event", "forwarded", "microsoft_dhcp"]
					}
				},
					"id": "winlog-windows.powershell-7886c582-c3fa-4ce1-a016-a6819dbba179",
					"enabled": true
				
			},
			{
				"data_stream": {
					"type": "logs",
					"dataset": "windows.powershell_operational"
				},
				"vars": {
					"period": {
						"type": "text",
						"value": "10s"
					},
					"preserve_original_event": {
						"type": "bool",
						"value": false
					},
					"event_id": {
						"type": "text",
						"value": "some_id"
					},
					"tags": {
						"type": "text",
						"value": ["preserve_original_event", "forwarded", "microsoft_dhcp"]
					}
				},
				"id": "winlog-windows.powershell_operational-7886c582-c3fa-4ce1-a016-a6819dbba179",
				"enabled": true
			},
			{
				"data_stream": {
					"type": "logs",
					"dataset": "windows.sysmon_operational"
				},
				"vars": {
					"period": {
						"type": "text",
						"value": "10s"
					},
					"preserve_original_event": {
						"type": "bool",
						"value": false
					},
					"event_id": {
						"type": "text",
						"value": "some_id"
					},
					"tags": {
						"type": "text",
						"value": ["preserve_original_event", "forwarded", "microsoft_dhcp"]
					}
				},
				"id": "winlog-windows.sysmon_operational-7886c582-c3fa-4ce1-a016-a6819dbba179",
				"enabled": true
			},
			{
				"data_stream": {
					"type": "metrics",
					"dataset": "windows.perfmon"
				},
				"vars": {
					"period": {
						"type": "text",
						"value": "10s"
					},
					"preserve_original_event": {
						"type": "bool",
						"value": false
					},
					"event_id": {
						"type": "text",
						"value": "some_id"
					},
					"tags": {
						"type": "text",
						"value": ["preserve_original_event", "forwarded", "microsoft_dhcp"]
					}
				},
				"id": "windows/metrics-windows.perfmon-7886c582-c3fa-4ce1-a016-a6819dbba179",
				"enabled": true
			},
			{
				"data_stream": {
					"type": "metrics",
					"dataset": "windows.service"
				},
				"vars": {
					"period": {
						"type": "text",
						"value": "10s"
					},
					"preserve_original_event": {
						"type": "bool",
						"value": false
					},
					"event_id": {
						"type": "text",
						"value": "some_id"
					},
					"tags": {
						"type": "text",
						"value": ["preserve_original_event", "forwarded", "microsoft_dhcp"]
					}
				},
				"id": "windows/metrics-windows.service-7886c582-c3fa-4ce1-a016-a6819dbba179",
				"enabled": true
			}
		]
	}]
}
```

**Pull request closed without merging**